fiftyone/server/routes/geo.py

```diff
@@ -69,8 +68,8 @@ async def _fetch_geo_points(
     results = {}
 
     async for result in agg_results:
-        if result and "sampleId" in result and "coordinates" in result:
-            results[result["sampleId"]] = result["coordinates"]
+        if result and "_id" in result and "coordinates" in result:
```

Comment:
you don't need this check. Since you're projecting the fields, they will always be there, even if null (but that doesn't seem to matter since you aren't checking the values here either)

Reply:
sure. will remove